### PR TITLE
Rename to COVID Green

### PR DESCRIPTION
**Pull request by dharding**
The purpose of this PR is to rename the project internals to align with the COVID Green name.

- The package/bundle identifiers for ios and android have been switched to `io.lfph.covidgreen`
- The XCode project files have all been updated to align with the COVID Green name
- Where necessary, small updates were made to make the app build on both platforms (e.g. updating gradle to work with newer JDKs)
- Documentation links now point to the covid green repo

### CODE REVIEW

**Review by colmharte — APPROVED** [review]

File: android/build.gradle [hunk 1]
```diff
@@ -13,8 +13,7 @@ buildscript {
         maven { url 'https://dl.bintray.com/android/android-tools/' }
     }
     dependencies {
-        classpath('com.android.tools.build:gradle:3.6.3')
-
+        classpath('com.android.tools.build:gradle:4.0.0')
```
Comment (jasnell) [inline android/build.gradle:16]: Unrelated change?

File: android/gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.0.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.5.1-all.zip
```
Comment (jasnell) [inline android/gradle/wrapper/gradle-wrapper.properties:3]: Unrelated change?

**Review by jasnell — APPROVED** [review]

File: android/build.gradle (hunk 1 above)
Comment (davidjharding) [reply android/build.gradle:16]: I was unable to build the app to confirm that the name updates did not break the build without this update. Newer versions of the JDK require an updated gradle.

File: android/gradle/wrapper/gradle-wrapper.properties (hunk 2 above)
Comment (davidjharding) [reply android/gradle/wrapper/gradle-wrapper.properties:3]: I was unable to build the app to confirm that the name updates did not break the build without this update. Newer versions of the JDK require an updated gradle.

Comment (jasnell) [reply android/gradle/wrapper/gradle-wrapper.properties:3]: Figured it was something along those lines
